test/resources/migrations/20211227102201-session.up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE sessions (
+    session_data json not null,
```

Comment:
is there a reason to you json instead of jsonb here?

Reply:
nope